packages/@react-stately/table/src/TableColumnLayout.ts

```diff
@@ -93,16 +93,16 @@ export class TableColumnLayout<T> {
       let frKey;
       let frValue;
       minWidths.set(column.key, this.getDefaultMinWidth(collection.columns[i]));
-      if (col !== column.key && !column.column.props.width && !isStatic(uncontrolledWidths.get(column.key))) {
+      if (col !== column.key && !column.props.width && !isStatic(uncontrolledWidths.get(column.key))) {
```

Comment:
These nodes are already columns, so no need to do `column.column`. RAC doesn't have these references either.

Reply:
I feel like Rob had a reason for `column.column`, trying to dig up the original reason.

Reply:
Can't seem to find anything in my digging. This should be fine since tests/functionality still passes, will ask him when he gets back.